### PR TITLE
removed spanish units in states they don't own

### PR DESCRIPTION
**Pull request by Emifas**

### CODE REVIEW

**Review by Danarca — CHANGES_REQUESTED** [review]
image
Missing an admiral for the navy, causing these errors; 

`[12:00:49][jomini_script_system.cpp:262]: Script system error!
  Error: fully_mobilize_army effect [ Scope formation cannot be mobilized ]
  Script location: file: common/history/military_deployments/00_imperia_military_deployments.txt line: 10`

`[12:00:49][jomini_script_system.cpp:262]: Script system error!
  Error: teleport_to_front effect [ Cannot teleport scope formation Real Armada EspaÃ±ola (92) to target front Toledo Navarra Front (0) due to:  ]
  Script location: file: common/history/military_deployments/00_imperia_military_deployments.txt line: 11`

In case there's still errors, try placing a `?` sign in front of the `=` in the country tag parts; so for example it might be `c:SPA ?= {`

**Review by Danarca — APPROVED** [review]
There we go, no more errors.

File: common/history/military_deployments/00_imperia_military_deployments.txt [hunk 1]
```diff
@@ -7,6 +7,9 @@
 	}
 	c:SPA = {
 		every_military_formation = {
+			limit = {
+				is_army = yes	
+			}
```
Comment (HiddeLekanne) [inline common/history/military_deployments/00_imperia_military_deployments.txt:12]: This is not valid code?

Comment (HiddeLekanne) [inline common/history/military_deployments/00_imperia_military_deployments.txt:12]: you want to still mobilize, just not teleported.

**Review by HiddeLekanne — CHANGES_REQUESTED** [review]

File: common/history/military_deployments/00_imperia_military_deployments.txt (hunk 1 above)
Comment (Danarca) [reply common/history/military_deployments/00_imperia_military_deployments.txt:12]: > you want to still mobilize, just not teleported.

This is vanilla behaviour though for conflicts ongoing at game start? Unless I'm misunderstanding :s

Comment (HiddeLekanne) [reply common/history/military_deployments/00_imperia_military_deployments.txt:12]: nvm I thought it was only teleporting.

**Review by HiddeLekanne — APPROVED** [review]

**Pull request merged**